transifex/push.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""
+This script can be used to automatically extract translation strings,
+commit, push, and merge them to their respective repos, and then push them to transifex.
+
+To use, export an environment variable `GITHUB_ACCESS_TOKEN`. The token requires
+GitHub's "repo" scope.
+
+Run the script from the root of this repo.
+
+    python3 transifex/push.py git@github.com:edx/course-discovery.git
```

Comment:
Following up on https://github.com/edx/ecommerce-scripts/pull/57#discussion_r123099995: you've already denoted the expected version of Python with the shebang line above. You'll typically see the expected version of Python aliased to `python` in most virtualenvs. Do whatever you're comfortable with here, but leaving it as `python` is sufficient.